src/services/events/events-router.ts

```diff
@@ -5,6 +5,42 @@ import { Database } from "../../database";
 
 const eventsRouter = Router();
 
+// Get current or next event based on current time
+eventsRouter.get("/currentOrNext", async (req, res, next) => {
+    const currentTime = new Date();
+
+    try {
+        const events = await Database.EVENTS.find().sort({ startTime: 1 });
+        let currentEvent = null;
+        let nextEvent = null;
+
+        for (const event of events) {
+            const startTime: Date = event.startTime as Date;
+            const endTime: Date = event.endTime as Date;
+
+            if (startTime <= currentTime && endTime >= currentTime) {
+                currentEvent = event;
+                break;
+            } else if (startTime > currentTime) {
+                nextEvent = event;
+                break;
+            }
+        }
```

Comment:
can this not all be cleaned up in one go: 

indOne({
    startTime: { $gte: now }
  }).sort({ startTime: 1 });